http/src/main/resources/init-resources/stratum-docker-compose.yaml

```diff
@@ -0,0 +1,7 @@
+version: '2'
```

Comment:
I had this as a separate docker-compose file but wondering if we should add it to `[jupyter|rstudio]-docker-compose.yaml` instead. Need to figure out how to make this container shut down the jupyter container